2020Robot/src/main/java/frc/robot/core751/subsystems/DifferentialDriveTrain.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.core751.subsystems;
+
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.core751.wrappers.WCANSparkMax;
+
+public class DifferentialDriveTrain extends SubsystemBase {
+
+    public enum driveMotor {
+        kSparkMaxBrushless;
+    }
+
+    private SpeedController[] leftArray;
+    private SpeedController[] rightArray;
+
+    private SpeedControllerGroup leftGroup;
+    private SpeedControllerGroup rightGroup;
+
+    private DifferentialDrive differentialDrive;
+
+    private static SpeedControllerGroup arrayToGroup(SpeedController[] sp) {
+        //There has to be a better way to do this
```

Comment:
I mean, if it works and isn't too ugly it works

Reply:
agreed